packages/lib/models/Tag.ts

```diff
@@ -216,4 +216,27 @@ export default class Tag extends BaseItem {
 			return tag;
 		});
 	}
+
+	static sortTags(tags: any) {
```

Comment:
Looks like you're using this function to compare objects of different types in different places: sometimes it's tags, other times it's `{ value: tag.id, label: tag.title }`
It seems to me it's best to rewrite it to always have args of the same time. Then you can also get rid of `any` and give it proper type.

Reply:
sortTags sorts on the basis of the title of the tag so it will work for `{ value: tag.id, label: tag.title }` and tags as well. I can write an if statement for it in sortTags but even without it the functionality will work.

Reply:
The issue with how it's written right now is that if someone wanted to use your function elsewhere, it's not possible to understand by looking at the signature what kind parameters it expects. You have to look at its implementation to find that it should be an array of things that have either title or label.

At the same time, you can make a trivial change to specify that this function requires an array of Tags and it makes things much more clear.

Reply:
For the type, it's possible to use TagEntity. Also I think this function shouldn't support sorting by label. Instead the view should be changed so instead of having Load tags => convert to list items => sort. You should have Load tags => sort => convert to list items. That way you don't need any special case in this function